cep-20.2.md

```diff
@@ -0,0 +1,831 @@
+
+  # settings concerning only Python
+  python:
+    # List of strings, where each string follows this format:
+    # PythonEntryPoint: `bsdiff4 = bsdiff4.cli:main_bsdiff4`
+    entry_points: [PythonEntryPoint]
+
+    # Specifies if python.app should be used as the entrypoint on macOS
+    # was `osx_is_app`
+    use_python_app_entrypoint: bool (defaults to false)  # macOS only!
+
+    # used on conda-forge, still needed?
+    preserve_egg_dir: bool (default false)
+
+    # skip compiling pyc for some files (was `skip_compile_pyc`)
+    skip_pyc_compilation: [glob]
+
+  # settings concerning the prefix detection in files
+  prefix_detection:
+    # force the file type of the given files to be TEXT or BINARY
+    # for prefix replacement
+    force_file_type:
+      # force TEXT file type
+      # was `???`
+      text: [glob]
+      # force binary file type
+      # was `???`
+      binary: [glob]
+
+    # ignore all or specific files for prefix replacement
+    # was `ignore_prefix_files`
+    ignore: bool | [path] (defaults to false)
+
+    # wether to detect binary files with prefix or not
+    # was `detect_binary_files_with_prefix`
+    ignore_binary_files: bool (defaults to true on Unix and (always) false on Windows)
+
+  # settings for shared libraries and executables
+  dynamic_linking:
+    # linux only, list of rpaths (was rpath)
+    rpaths: [path] (defaults to ['lib/'])
+
+    # wether to relocate binaries or not. If this is a list of paths, then
+    # only the listed paths are relocated
+    binary_relocation: bool (defaults to true) | [glob]
+
+    # Allow linking against libraries that are not in the run requirements
+    # (was `missing_dso_whitelist`)
+    missing_dso_allowlist: [glob]
+
+    # Allow runpath / rpath to point to these locations outside of the environment
+    # (was `runpath_whitelist`)
+    rpath_allowlist: [glob]
+
+    # what to do when detecting overdepending
+    overdepending_behavior: OneOf<"ignore" | "error"> # (defaults to "error")
+
+    # what to do when detecting overlinking
+    overlinking_behavior: OneOf<"ignore" | "error"> # (defaults to "error")
+
+  # REMOVED:
+  # pre-link: string (was deprecated for a long time)
+  # Whether to include the recipe or not in the final package - should be specified on command line or other config file?
+  # noarch_python: bool
+  # features: list
+  # msvc_compiler: str
+  # requires_features: dict
+  # provides_features: dict
+  # preferred_env: str
+  # preferred_env_executable_paths: list
+  # disable_pip: bool
+  # marked as "still experimental"
+  # pin_depends: Enum<"record" | "strict">
+  # overlinking_ignore_patterns: [glob]
+  # defaults to patchelf (only cudatoolkit is using `lief` for some reason)
+  # rpaths_patcher: None
+  # post-link: path
+  # pre-unlink: path
+  # pre-link: path
+```
+
+### Script section
+
+```yaml
+script:
+  # the interpreter to use for the script
+  interpreter: string  # defaults to bash on UNIX and cmd.exe on Windows
```

Comment:
Is the intention here to force conda build (or equivalent) to explicitly invoke `bash`, or are we going to assume that `/bin/sh` or `/usr/bin/env sh` is actually bash? (Moving forward, I'm not sure it's safe to assume `sh` and `bash` are equivalent.)